applications/nrf5340_audio/src/bluetooth/le_audio_bis_gateway.c

```diff
@@ -234,8 +234,8 @@ static int adv_create(void)
 		broadcast_id = CONFIG_BT_AUDIO_BROADCAST_ID_FIXED;
 	}
 
-	ext_ad[0] = (struct bt_data)BT_DATA_BYTES(BT_DATA_BROADCAST_NAME,
-						  CONFIG_BT_AUDIO_BROADCAST_NAME);
+	ext_ad[0] = (struct bt_data)BT_DATA(BT_DATA_BROADCAST_NAME, CONFIG_BT_AUDIO_BROADCAST_NAME,
```

Comment:
This change is fine, but may this apply in other places as well?

Reply:
Broadcast name isn't used anywhere else, AFAICT no string are added in a similar matter - but there might very well be strings I'm not aware of